lib/internal/loader/Loader.js

```diff
@@ -91,7 +91,7 @@ class Loader {
       throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'url', 'string');
     }
 
-    if (format === 'builtin') {
+    if (format === 'builtin/esm' || format === 'builtin/cjs') {
```

Comment:
Why do we need two forms of builtin? Can the `builtin/esm` not entirely replace the `builtin/cjs`? What is the use case for retaining `builtin/cjs`?